src/API.jl

```diff
@@ -1329,7 +1352,12 @@ function precompile(ctx::Context, pkgs::Vector{PackageSpec}; internal_call::Bool
                                 # If not offset, on larger terminal fonts it looks odd that they all sync-up
                                 anim_char = anim_chars[(i + Int(dep.name[1])) % length(anim_chars) + 1]
                                 anim_char_colored = dep in direct_deps ? anim_char : color_string(anim_char, :light_black)
-                                print(iostr, "  $anim_char_colored $name\n")
+                                waiting = if dep in taskwaiting
+                                    color_string(" Waiting for trailing task to finish. Use ctrl-c to interrupt and inspect warnings", Base.warn_color())
```

Comment:
The word `trailing` does not convey a clear meaning to me. Maybe 'background' or 'async'? Perhaps saying 'background task, IO, or timer to finish` in full would also be even clearer?

Reply:
`background task, IO, or timer to finish` sounds good